lab_1/main.py

```diff
@@ -198,8 +158,41 @@ def get_adjacent_words(tokens: list, word: str, left_n: int, right_n: int) -> li
     right_n = 3
     --> [['man', 'is'], ['dog, 'cat']]
     """
-    pass
-
+    adjacent_words = []
+    if type(tokens) != list or type(word) != str:
+        return []
+    elif type(left_n) != int or type(right_n) != int:
+        return []
+    elif left_n < 1 and right_n < 1:
+        return []
+    concordance = get_concordance (tokens, word, left_n, right_n)
+    for usage in concordance:
+        pair_of_words = []
+        for number, token in enumerate(usage):
+            if token == word:
+                if left_n >= 1 and right_n < 1:
+                    if left_n > len(usage[:number]):
```

Comment:
do you need these checks at all?
you have already sent left_n and right_n to "get_concordance" so that these variables are  supposed to be already checked

Reply:
I need these checks to get certain words from left and right, but I deleted the type-checks in the beginning)